tests/drivers/st77xx/Makefile

```diff
@@ -2,7 +2,7 @@ BOARD ?= stm32f429i-disc1
 
 include ../Makefile.drivers_common
 
-USEMODULE += st7735
+USEMODULE += st77xx
```

Comment:
I don't like that way of doing the dependency resolution. I would rather do:

```mk
DRIVER ?= st7735
USEMODULE += $(DRIVER)
```

Reply:
See my comment above.